packages/react/src/Breadcrumbs/Breadcrumbs.module.css

```diff
@@ -153,6 +136,7 @@
     & summary {
       list-style: none;
       cursor: pointer;
+      outline: none;
```

Comment:
This makes the focus outline around the menu button go away.  Since button + summary is same element. When you tab along, there is no focus outline.